package-lock.json

```diff
@@ -3352,9 +3352,9 @@
       }
     },
     "underscore": {
-      "version": "1.9.1",
-      "resolved": "https://registry.npmjs.org/underscore/-/underscore-1.9.1.tgz",
-      "integrity": "sha512-5/4etnCkd9c8gwgowi5/om/mYO5ajCaOgdzj/oW+0eQV9WxKBDZw5+ycmKmeaTXjInS/W0BzpGLo2xR2aBwZdg=="
+      "version": "1.11.0",
```

Comment:
Is there a reason we need to upgrade `underscore`?

Reply:
I thought there was, but looking again. No. Reverted.